src/index.ts

```diff
@@ -282,6 +282,10 @@ async function activate(
   // or are unable to use it.
   let hideClusterManager: boolean = false;
 
+  // Whether to test the Dask dashboard using a fetch request or to proceed
+  // with default behavior.
+  let enableTestDashboardFetch: boolean = false;
```

Comment:
Further down the file (~line 345) there is an `onSettingsChanged` callack which is called whenever the setting for this is updated. That's where you would go to the `sidebar` and update it to either make the browser check or the server-side check.

I described sending that data in using an option in the `IOptions`object, but on reflection it could also just be a setter on the object itself. As an example, you might look at the `active` getter/setter on the `DaskDashboard` class.